torch/onnx/symbolic_opset14.py

```diff
@@ -117,3 +119,174 @@ def quantized_hardswish(g: jit_utils.GraphContext, x, op_scale, op_zero_point):
     output = hardswish(g, x)
 
     return symbolic_helper.quantize_helper(g, output, op_scale, op_zero_point)
+
+
+# Ported from https://github.com/microsoft/onnx-script/blob/main/onnxscript/function_libs/torch_aten/ops/core.py
+# aten_scaled_dot_product_attention
+# NOTE: Need Trilu
+@_onnx_symbolic("aten::scaled_dot_product_attention")
+@symbolic_helper.parse_args("v", "v", "v", "v", "f", "b", "f")
+@_beartype.beartype
+def scaled_dot_product_attention(
+    g: jit_utils.GraphContext,
+    query: torch._C.Value,
+    key: torch._C.Value,
+    value: torch._C.Value,
+    attn_mask: Optional[torch._C.Value] = None,
+    dropout_p: float = 0.0,
+    is_causal: bool = False,
+    scale: Optional[float] = None,
```

Comment:
Does it fail when scale is None, since it is "f"?

Reply:
I don't get it. Could you elaborate?

Reply:
You mean in 
```python
if scale is None:
```

Reply:
So in the decorator `@symbolic_helper.parse_args("v", "v", "v", "v", "f", "b", "f")` scale is the last, which is `f`. I was just wondering if scale were `None`, whether `parse_args` will fail with an error saying it cannot parse the arg.

Reply:
Done